dev/NavigationView/NavigationView.cpp

```diff
@@ -616,6 +784,15 @@ bool NavigationView::IsRootItemsRepeater(const winrt::DependencyObject& element)
     return false;
 }
 
+bool NavigationView::IsRootGridOfFlyout(const winrt::DependencyObject& element)
+{
+    if (auto grid = element.try_as<winrt::Grid>())
+    {
+        return grid.Name() == c_flyoutRootGrid;
```

Comment:
>return grid.Name() == c_flyoutRootGrid; [](start = 8, length = 39)

I still don't like this, Why do we need to compare by name?

Reply:
@ojhad Please create an issue and link it here.

Reply:
https://github.com/microsoft/microsoft-ui-xaml/issues/2173